src/components/Avatar/Avatar.scss

```diff
@@ -0,0 +1,20 @@
+@import "~styles/main";
+
+.container {
+  img {
+    border-radius: 50%;
+    width: 50px;
+    height: 50px; //70px
```

Comment:
Can you explain why height is specified as 50px but also commented as 70px?

Reply:
In templates, there are 4 varying sizes for avatar- 2 each are of similar size (70 67, 48 50)
So I thought to make it conditional or use modifiers to display 2 variants. Comment serves for reminder for me

Reply:
Done